peps/pep-0772.rst

```diff
@@ -0,0 +1,292 @@
+
+Each council member's term will be two (2) years, from when their elections are finalised until the next elections for their cohort are finalised.
+
+Only for elections involving the entire council (like the initial council election), the two candidates receiving the highest number of votes shall be designated Cohort A with a two (2) year term, and the three candidates receiving the highest number of votes after shall be designated Cohort B with a one (1) year term.
+
+There are no term limits for individual council members.
+
+Vacancies
+---------
+
+Council members may resign their position at any time.
+
+Whenever there is a vacancy during the regular council term, the council may vote to appoint a replacement to serve out the rest of the term.
+
+If a council member drops out of touch and cannot be contacted for a month or longer, then the rest of the council may vote to replace them (with a simple majority vote, not counting the relevant member).
+
+Conflicts of interest
+---------------------
+
+No more than two Packaging Council members should be employed by or significantly affiliated with the same entity.
+
+While we trust council members to act in the best interests of Python rather than themselves or their employers, the mere appearance of any one company dominating Python development could itself be harmful and erode trust.
+
+In a council election, if more than 2 of the top 5 vote-getters work for the same employer, then whichever of them ranked lowest is disqualified and the 6th-ranking candidate moves up into 5th place; this is repeated until a valid council is formed.
+
+During a council term, if changing circumstances cause this rule to be broken (for instance, due to a council member changing employment), then one or more council members must resign to remedy the issue, and the resulting vacancies can then be filled as normal.
+
+-------------------
+Packaging community
+-------------------
+
+Responsibility
+==============
+
+Packaging community members participate in formal votes to elect the Packaging Council.
+
+Processes
+=========
+
+Initial membership
+------------------
+
+Initial membership in the Packaging community will include anyone who has taken the time to formalise their participation in the Packaging community. This includes:
+
+* PyPA members: Anyone with the triage bit or commit bit, or at least one project in the PyPA organisation.
+* Packaging workgroup members: Anyone who is listed on the Packaging WG charter will be moved into the Packaging Community.
+* Interested core team members: Any Python core team member who is willing to participate is welcome.
+* Wider community members: Non-profit organisations that participate in packaging or working with new packagers. For example, PyOpenSci, NumFocus, Django, are encouraged to initially nominate up to 7 members by sending an email to \[todo\].
```

Comment:
```suggestion
* Wider community members: non-profit organisations that participate in packaging or working with new packagers. For example, PyOpenSci, NumFocus, Django, are encouraged to initially nominate up to 7 members by sending an email to \[todo\].
```

Reply:
The `:` is acting as a separator and all other entries in this list use a capitalized word after it. I'd prefer to keep the current style here.